recipes-core/psplash/psplash/psplash-start.service

```diff
@@ -5,7 +5,7 @@ After=systemd-vconsole-setup.service systemd-udev-trigger.service systemd-udevd.
 DefaultDependencies=no
 
 [Service]
-ExecStart=/usr/bin/psplash --angle 270
+ExecStart=/usr/bin/psplash
```

Comment:
OK to remove because you rotated the image itself?

Reply:
The image is symetrical. It looks identical from all sides.

Reply:
Still testing though ... plz wait with merging

Reply:
Commit message updated with this info so resolving.